pkg/report/adocReport.go

```diff
@@ -310,9 +311,11 @@ func (adoc adocReport) WriteReport(model *types.Model,
 	if err != nil {
 		return fmt.Errorf("error creating shared runtimes: %w", err)
 	}
-	err = adoc.writeRiskRulesChecked(modelFilename, skipRiskRules, buildTimestamp, threagileVersion, modelHash, customRiskRules)
-	if err != nil {
-		return fmt.Errorf("error creating risk rules checked: %w", err)
+	if val, ok := showChapters[RiskRulesCheckedByThreagile]; !ok || val {
```

Comment:
I'm getting an impression that by default we're not showing chapters and I think it would be better to by default show chapters and allow user to hide unwanted chapters. 
Is it only related to adoc? Or is it applicable for pdf report as well?

Reply:
changed the whole naming from show to hide, perhaps now its more clear that it is shown by default? The change to the test is not relevant, but since its flaky and from now and then it fails for my commits, which lead to an unhealthy red cross in pull requests i fixed it as well.

Reply:
thanks